### PR TITLE
feat(woostack-plan): gate §7 acceptance-criteria breadth in self-review

### PR DESCRIPTION
**Pull request by howarewoo**
## Goal

Close the breadth loop opened by #247: make the plan engine *enforce* that the spec's §7 acceptance criteria — and their happy/error/edge cases — actually land as TDD tasks/tests, so structured breadth isn't just authored, it's gated. Increment 2 of 2.

## Summary

- `woostack-plan/SKILL.md` self-review step 1 now asserts **AC coverage**: every spec §7 AC (and each filled, non-N/A happy/error/edge case) maps to a task/test; a whole-section `N/A` is sanity-checked against the spec body (else flagged suspect).
- Cross-references the banned-placeholder rule ("handle edge cases") to §7 as the structured home for error/edge cases — keeps the ban, gives it a destination.
- `plan-template.md` self-review checklist gains a matching **AC coverage** line.
- Plan task mechanics unchanged — the gate checks test *existence*, not task count.

### Automated

- [x] `grep`: SKILL.md carries `**AC coverage:**`, the `no behavioral requirement` sanity-check, and the §7 cross-reference — **PASS**
- [x] `grep`: plan-template.md self-review checklist carries the `**AC coverage**` line — **PASS**
- [x] Both red→green transitions observed (assertions failed pre-edit, passed post-edit).

### Manual

**Before merge**

- [ ] Read the `woostack-plan/SKILL.md` self-review diff + `plan-template.md` checklist line — confirm the AC-coverage + N/A-sanity wording is clear and the §7 cross-ref reads right.

Spec: .woostack/specs/2026-06-06-spec-acceptance-criteria.md

### CODE REVIEW

**Comment by howarewoo** [comment]
* **#248** <a href="https://app.graphite.com/github/pr/howarewoo/woostack/248?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.com/github/pr/howarewoo/woostack/248?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#247** <a href="https://app.graphite.com/github/pr/howarewoo/woostack/247?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#246** <a href="https://app.graphite.com/github/pr/howarewoo/woostack/246?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: skills/woostack-plan/SKILL.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Every step carries the actual content an engineer needs. These are plan failures
 them:
 
 - "TBD", "TODO", "implement later", "fill in details"
-- "Add error handling" / "add validation" / "handle edge cases"
+- "Add error handling" / "add validation" / "handle edge cases" — write the actual test instead; error and edge cases belong in the spec's §7 Acceptance criteria, enumerated there as happy/error/edge
```
Comment (howarewoo) [inline skills/woostack-plan/SKILL.md]: **Nit: Truncated phrase — missing "cases"**

The added clause ends with "enumerated there as happy/error/edge" — `cases` is missing, so the list reads as incomplete. Every other occurrence (plan-template.md, SKILL.md:133) uses the full "happy/error/edge cases".

```suggestion
- "Add error handling" / "add validation" / "handle edge cases" — write the actual test instead; error and edge cases belong in the spec's §7 Acceptance criteria, enumerated there as happy/error/edge cases
```

· NIT

**Review by howarewoo — COMMENTED** [review]
## 🤖 woostack-review — ✅ APPROVED WITH SUGGESTIONS (fast)

No blocking findings. 5 angles ran at fast tier — `bugs`, `security`, `skills`, `docs`, `aeo` — receipt gate 5/5. Adversarial validators: 4 raw → **1 kept** (1 nit). Dropped a casing nit and a structural-parallelism nit as style-only.

1 non-blocking **nit** inline (a one-word truncation in the new self-review clause).

_Self-authored PR → event downgraded to COMMENT (GitHub blocks self-APPROVE); the verdict above is the real result._

File: skills/woostack-plan/SKILL.md (hunk 1 above)
Comment (howarewoo) [reply skills/woostack-plan/SKILL.md]: Fixed in 8ff7ac3 — appended "cases" so the clause reads "…enumerated there as happy/error/edge cases", matching plan-template.md and SKILL.md:133.

**Comment by howarewoo** [comment]
### Merge activity

* **Jun 7, 1:16 AM UTC**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.com/github/pr/howarewoo/woostack/248).
* **Jun 7, 1:19 AM UTC**: [Graphite](https://app.graphite.com/github/pr/howarewoo/woostack/248) rebased this pull request as part of a merge.
* **Jun 7, 1:20 AM UTC**: @howarewoo merged this pull request with [Graphite](https://app.graphite.com/github/pr/howarewoo/woostack/248).
